client/my-sites/importer/style.scss

```diff
@@ -38,6 +38,10 @@
 	&.wordpress {
 		background-color: $blue-wordpress;
 	}
+
+	&.medium {
+		background-color: darken( $gray-light, 31% );
```

Comment:
The values around 30% looked best to me, but there is no specific reason for using 31%. I just happen to like prime numbers. :D